chainer/links/connection/gru.py

```diff
@@ -185,21 +185,20 @@ def __call__(self, x):
 class GRU(StatefulGRU):
     """Stateful Gated Recurrent Unit function (GRU)
 
-    This is an alias of "~chainer.links.StatefulGRU".
-    Its documented API is identical to the function.
+    This is an alias of :class:`~chainer.links.StatefulGRU`.
 
     .. warning::
 
        In Chainer v1, :class:`~chainer.links.GRU` was *stateless*,
```

Comment:
- ``` :class:`~chainer.links.StatefulGRU` ``` does not generate a link. That's because there's no entry of `StatefulGRU` in `docs/source/reference/links.rst`. Can you add to it?